models/index.js

```diff
@@ -0,0 +1,32 @@
+const User = require('./User');
+const Hike = require('./Hike');
+const Trail = require('./Trail');
+const Comment = require('./Comment');
+
+//relations
+//1 user can have many hikes
+//1 user can have many comments
+//1 trail can have many hikes
+//1 trail can have many comments
+
```

Comment:
I think we can also include the corresponding "belongsTo" relations for the modes. e.g.
`User.hasMany(Hike, {...})`
`Hike.belongsTo(User, {...})`